pkgs/os-specific/linux/zfs/default.nix

```
@@ -17,6 +17,7 @@
 
 # for determining the latest compatible linuxPackages
 , linuxPackages_5_19 ? pkgs.linuxKernel.packages.linux_5_19
+, linuxPackages_6_0 ? pkgs.linuxKernel.packages.linux_5_20
```

Why did you change this to `linux_5_20`? That attribute doesn't even exist.
```suggestion
, linuxPackages_6_0 ? pkgs.linuxKernel.packages.linux_6_0
```

---

Ouch. Good catch.

---

I'm overriding this locally on my machine and *was* testing against 6.0.x. After a reboot:

```console
$ uname -srv                                                                                                                                                                               
Linux 6.0.5 #1-NixOS SMP PREEMPT_DYNAMIC Wed Oct 26 10:53:32 UTC 2022
```